include/driver_entry_interface.h

```diff
@@ -210,7 +200,7 @@ DEFINE_GUID(GUID_DEVINTERFACE_bareflank,
  * @param arg character buffer containing the module to add
  * @return
  */
-#define IOCTL_ADD_MODULE CTL_CODE(BAREFLANK_DEVICETYPE, 0x901, METHOD_IN_DIRECT, FILE_WRITE_DATA)
+#define IOCTL_ADD_MODULE CTL_CODE(BAREFLANK_DEVICETYPE, IOCTL_ADD_MODULE_CMD, METHOD_IN_DIRECT, FILE_WRITE_DATA)
```

Comment:
These values are reserved for Microsoft. 

>FunctionCode

Identifies the function to be performed by the driver. Values of less than 0x800 are reserved for Microsoft. Values of 0x800 and higher can be used by vendors. Note that the vendor-assigned values set the Custom bit.https://msdn.microsoft.com/en-us/library/windows/hardware/ff543023(v=vs.85).aspx